.eslintrc

```
@@ -10,15 +10,21 @@
   },
   "rules": {
     "indent": ["warn", "tab"],
```

Maybe override vue/html-indent (defaults to spaces) to tabs so it matches our rule for indenting js.
```suggestion
    "indent": ["warn", "tab"],
    "vue/html-indent": ["warn", "tab"],
```